QStreak/View Controllers/Record List/RecordListViewController.swift

```diff
@@ -55,4 +55,10 @@ extension RecordListViewController: UITableViewDataSource, UITableViewDelegate {
         cell.detailTextLabel?.text = "\(viewModel.records[indexPath.row].creationDate)"
         return cell
     }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let destination = segue.destination as? RecordDetailViewController {
+            destination.recordToEdit = viewModel.records[(tableView.indexPathForSelectedRow?.row)!]
```

Comment:
I'm not sure this is the best way of going about this. I'm guessing I should be doing something with the `DetailViewModel` class but not actually sure what...some pointers would be helpful if this isn't the convention that I should be following.

Reply:
Instead of relying on the storyboard to do the transition, could use the tableView's `didSelectRowAt` delegate method. Use the `indexPath` to get the right record and inject like you're doing here. Instantiate the new view controller (there's an example in `AccountSetupViewController`'s `continueButtonTapped`). Then use the `navigationController` property to push the new screen.